tests/std/tests/P0896R4_ranges_alg_uninitialized_fill_n/test.cpp

```diff
@@ -120,6 +120,20 @@ struct throwing_test {
     }
 };
 
+struct memset_test {
+    static constexpr int expected[] = {42, 42, 42};

```

Comment:
Same comment, change `int` to `unsigned char` or another 1-byte type here to exercise `memset`.